.codecov.yml

```diff
@@ -0,0 +1,26 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+codecov:
+  branch: master
```

Comment:
We should run this in 3.x too.

Reply:
So please move this PR to the 3.x branch.

Reply:
> We should run this in 3.x too

From my understanding, the `branch` option in the .yaml is used to specify which branch is displayed by default on the codecov project homepage. From [codecov documentation](https://docs.codecov.io/v4.3.6/docs/codecov-yaml#section-default-branch):

"This will change the branch used on the default project dashboard"

I think that master is used by default, so I've only kept this option in the .yaml file in case we want to change it later. Do you prefer to have the 3.x instead?

> So please move this PR to the 3.x branch.

You are right, I will rebase this PR to the 3.x branch.